app/(tabs)/account/Account.tsx

```diff
@@ -0,0 +1,287 @@
+import React, { useState } from "react";
+import {
+  ScrollView,
+  Switch,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+import { Ionicons } from "@expo/vector-icons";
+
+import { Button, Section, Slider } from "tamagui";
+
+import { commonStyles as styles } from "./styles";
+
+interface AccountProps {
+  username: string;
+  email: string;
+  password: string;
+  setUsername: (username: string) => void;
+  setEmail: (email: string) => void;
+  setPassword: (password: string) => void;
+  setIsLoggedIn: (isLoggedIn: boolean) => void;
+}
+
+export default function Account({
+  username,
+  email,
+  password,
+  setUsername,
+  setEmail,
+  setPassword,
+  setIsLoggedIn,
+}: AccountProps) {
+  const [selectedTab, setSelectedTab] = useState("Membership");
+  const [password2, setPassword2] = useState<string>("");
+  const [doNotDisturb, setDoNotDisturb] = useState<boolean>(false);
+  const [radius, setRadius] = useState<number>(50);
+  const [maxNotifications, setMaxNotifications] = useState<number>(10);
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+  };
+
+  const membershipOptions = [
+    {
+      name: "Freemium",
+      price: "7 days free trial!",
+      features: ["No Ads", "Store Preferences", "Unlocked Discounts"],
+    },
+    {
+      name: "Monthly",
+      price: "$6.99",
+      features: ["No Ads", "Store Preferences", "Unlocked Discounts"],
+    },
+    {
+      name: "Yearly",
+      price: "$69.99",
+      features: ["No Ads", "Store Preferences", "Unlocked Discounts"],
+    },
+  ];
+
+  const renderSection = () => {
+    switch (selectedTab) {
+      case "Membership":
+        return (
+          <View style={styles.wrapSubL}>
+            <Text style={styles.headerA}>Membership Section</Text>
+            <Text style={styles.inputTitleL}>
+              You do not have a subscriptions. Register for the free trial
+              before you miss out on these hard-to-beat features!
+            </Text>
+            <ScrollView
+              horizontal
+              showsHorizontalScrollIndicator={false}
+              style={styles.membershipSlider}
+              contentContainerStyle={styles.membershipContentContainer}
+            >
+              {membershipOptions.map((option, index) => (
+                <View key={index} style={styles.card}>
+                  <View style={styles.featureHeader}>
+                    <Text style={styles.cardText}>{option.name}</Text>
+                    <Text style={styles.cardPrice}>{option.price}</Text>
+                  </View>
+                  <View style={styles.featureList}>
+                    {option.features.map((feature, idx) => (
```

Comment:
If you need two separate variable names for indicies, I would suggest using i, j, k, etc
'idx' could be confused for 'ID-x'